src/Sdk/StrixMusic.Sdk.WinUI/Controls/Collections/AlbumCollection.cs

```diff
@@ -82,12 +82,12 @@ private void DetachHandlers()
         /// Dependency property for <ses cref="IAlbumCollectionViewModel" />.
         /// </summary>
         public static readonly DependencyProperty CollectionProperty =
-            DependencyProperty.Register(nameof(Collection), typeof(IAlbumCollectionViewModel), typeof(AlbumCollection), new PropertyMetadata(null, (s, e) => s.Cast<AlbumCollection>().OnCollectionChanged((IAlbumCollectionViewModel?)e.OldValue, (IAlbumCollectionViewModel?)e.NewValue)));
+            DependencyProperty.Register(nameof(Collection), typeof(IAlbumCollectionViewModel), typeof(AlbumCollection), new PropertyMetadata(null, (s, e) => s.Cast<AlbumCollection>().OnCollectionChangedAsync((IAlbumCollectionViewModel?)e.OldValue, (IAlbumCollectionViewModel?)e.NewValue)));
 
         /// <summary>
         /// Fires when the <see cref="Collection"/> property changes.
         /// </summary>
-        protected virtual void OnCollectionChanged(IAlbumCollectionViewModel? oldValue, IAlbumCollectionViewModel? newValue)
+        protected virtual void OnCollectionChangedAsync(IAlbumCollectionViewModel? oldValue, IAlbumCollectionViewModel? newValue)
```

Comment:
This method was renamed to async, but doesn't do anything async. Let's undo this.